samples/features/sql-big-data-cluster/private-bdc/README.md

```diff
@@ -0,0 +1,10 @@
+# Deploy BDC in private AKS cluster with User-defined Route (UDR)
```

Comment:
Please add the new folder under the already existing "deployment" folder since it falls under the same topic. Also include updates to the README.md file under the the "deployment topic.